### PR TITLE
Feature/initial api

### PR DESCRIPTION
**Pull request by annashamray**
Changes:
1. Create `Klant` resource 
1. Create `ContactMoment` resources

Issues:
1. https://github.com/VNG-Realisatie/gemma-zaken/issues/7 (Create API)
1. https://github.com/VNG-Realisatie/gemma-zaken/issues/1308 (add `ContactMoment.kanaal` field)
1. https://github.com/VNG-Realisatie/gemma-zaken/issues/1307 (add `ContactMoment.text` field)
1. https://github.com/VNG-Realisatie/gemma-zaken/issues/1309 (add `ContactMoment.datumtijd` field)
1. https://github.com/VNG-Realisatie/gemma-zaken/issues/1317 (add `ContactMoment.initiatiefnemer` field)
1. https://github.com/VNG-Realisatie/gemma-zaken/issues/1318 (make `ContactMoment.klant` an optional field)
1. https://github.com/VNG-Realisatie/gemma-zaken/issues/1323 (add `ContactMoment.zaak` field) 
1. https://github.com/VNG-Realisatie/gemma-zaken/issues/1321 (add `Klant` attributes: `voornaam`, `achternaam`, `adres`, `telefoonnummer`, `emailadres`)

### CODE REVIEW

File: src/kcc/api/serializers.py [hunk 1]
```diff
@@ -2,12 +2,38 @@
...
+class ContactMomentSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = ContactMoment
+        fields = (
+            "url",
+            "klant",
+            "zaak",
+            "datumtijd",
+            "kanaal",
+            "text",
```
Comment (joeribekker) [inline src/kcc/api/serializers.py]: Please rename to "tekst"

File: src/kcc/api/serializers.py [hunk 2]
```diff
@@ -2,12 +2,38 @@
...
+        )
+        extra_kwargs = {"url": {"lookup_field": "uuid"}}
+
+
+class ContactMomentSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = ContactMoment
+        fields = (
+            "url",
+            "klant",
```
Comment (joeribekker) [inline src/kcc/api/serializers.py:29]: Klant should have the usual help text that goes for relations:

URL-referentie naar een KLANT ...

File: src/kcc/api/viewsets.py [hunk 3]
```diff
@@ -1,41 +1,104 @@
...
 logger = logging.getLogger(__name__)
 
 
-class ExampleViewSet(viewsets.ModelViewSet):
+class KlantViewSet(viewsets.ModelViewSet):
     """
-    Describe viewset.
+    Opvragen en bewerken van KLANTen.
 
     create:
```
Comment (joeribekker) [inline src/kcc/api/viewsets.py:23]: retrieve description missing

File: src/kcc/api/viewsets.py [hunk 4]
```diff
@@ -1,41 +1,104 @@
...
+        "partial_update": SCOPE_KLANTEN_BIJWERKEN,
+        "destroy": SCOPE_KLANTEN_ALLES_VERWIJDEREN,
+    }
+
+
+class ContactMomentViewSet(viewsets.ModelViewSet):
+    """
+    Opvragen en bewerken van CONTACTMOMENTen.
+
+    create:
```
Comment (joeribekker) [inline src/kcc/api/viewsets.py:72]: retrieve description missing.

File: src/kcc/datamodel/models.py [hunk 5]
```diff
@@ -1,4 +1,50 @@
...
+        max_length=50,
+        help_text=_("Het communicatiekanaal waarlangs het CONTACTMOMENT gevoerd wordt"),
+    )
+    text = models.TextField(
+        blank=True,
+        help_text=_(
+            "Een toelichting die inhoudelijk het contact met de klant beschrijft."
+        ),
+    )
+    initiatiefnemer = models.CharField(
```
Comment (joeribekker) [inline src/kcc/datamodel/models.py:54]: missing help text:

"De partij die het contact heeft geïnitieerd."

**Review by joeribekker — CHANGES_REQUESTED** [review]
Looking good!

File: src/kcc/api/serializers.py [hunk 6]
```diff
@@ -2,12 +2,38 @@
...
+            "zaak",
+            "datumtijd",
+            "kanaal",
+            "text",
+            "initiatiefnemer",
```
Comment (joeribekker) [inline src/kcc/api/serializers.py:34]: This one "InitiatiefNemer" seems missing in the API spec.

File: src/kcc/api/serializers.py (hunk 1 above)
Comment (annashamray) [reply src/kcc/api/serializers.py]: done

File: src/kcc/api/viewsets.py (hunk 3 above)
Comment (annashamray) [reply src/kcc/api/viewsets.py:23]: "retrieve" description is added

File: src/kcc/api/viewsets.py (hunk 4 above)
Comment (annashamray) [reply src/kcc/api/viewsets.py:72]: "retrieve" description is added

File: src/kcc/datamodel/models.py (hunk 5 above)
Comment (annashamray) [reply src/kcc/datamodel/models.py:54]: help_text for `initiatiefnemer` is added

File: src/kcc/api/serializers.py (hunk 2 above)
Comment (annashamray) [reply src/kcc/api/serializers.py:29]: help_text for `klant` is added

File: src/kcc/api/serializers.py (hunk 6 above)
Comment (annashamray) [reply src/kcc/api/serializers.py:34]: API spec is regenerated, thanks for noticing

**Review by joeribekker — APPROVED** [review]

**Pull request merged**